openapi/schema_test.go

```diff
@@ -5,13 +5,13 @@ import (
 	"testing"
 )
 
-var (
-	properties = map[string]*Schema{
+func emptySchemaWith(childProperties, childItems, parent bool) *Schema {
```

Comment:
Consider functional option parameters here. They provide a less rigid way to initialise a Schema.

Reply:
I'll give it a bash, but I'm unclear in how that might look

Reply:
No biggie.